packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -24,6 +24,10 @@ typedef StreamChannel<String> _OpenChannel(Uri uri);
 
 _OpenChannel _openChannel = _defaultOpenChannel;
 
+/// A function that reacts to the invocation of the 'reloadSources' service.
+typedef Future ReloadSources(String isolateId,
```

Comment:
Seems like this is `Future<Null>` (not `Future<dynamic>`).  Either way, please update it to be explicit.